telemetry.go

```diff
@@ -40,8 +44,8 @@ var (
 		[]string{"outcome"},
 	)
 	mappingsCount = prometheus.NewGauge(prometheus.GaugeOpts{
-		Name: "statsd_exporter_loaded_mappings_count",
-		Help: "The number of configured metric mappings.",
+		Name: "statsd_exporter_loaded_mappings",
```

Comment:
While this is a better metric name and I support it, can we keep this breaking unrelated change out of this PR? Otherwise we will probably not catch it for the Changelog entry and all.